src/app/views/user/user-settings/user-profile/user-profile.component.html

```diff
@@ -85,18 +123,43 @@
         <label *ngIf="profileForm.value.current_status === 'asylum_seeker'" for="ssn">{{'user.profile.form.coordination.number.label' | translate}}:</label>
         <label *ngIf="profileForm.value.current_status === 'eu_citizen'" for="ssn">{{'user.profile.form.coordination.number.or.ssn.label' | translate}}:</label>
         <label *ngIf="profileForm.value.current_status !== 'asylum_seeker' && profileForm.value.current_status !== 'eu_citizen'" for="ssn">{{'user.profile.form.ssn.label' | translate}}:</label>
-        <input *ngIf="profileForm.value.current_status === 'asylum_seeker'" maxlength="20" class="profile-form-input-text form-control material-input" id="ssn" name="ssn-input" type="text" onkeypress="return event.charCode >= 48 && event.charCode <= 57" (focus)="serverValidationErrors.ssn = ''" placeholder="{{'user.profile.form.coordination.number.placeholder' | translate}}"
+        <input *ngIf="profileForm.value.current_status === 'asylum_seeker'"
+            maxlength="20" class="profile-form-input-text form-control material-input" id="ssn"
+            name="ssn-input"
+            type="text"
+            onkeypress="return event.charCode >= 48 && event.charCode <= 57"
```

Comment:
Is there any possibility that we can have these named somehow? (especially since we're using this in multiple places).

I had to google to find this table:
<img width="477" alt="screen shot 2017-02-21 at 20 45 06" src="https://cloud.githubusercontent.com/assets/922411/23181798/aa1d4028-f876-11e6-8343-777eb8500f2c.png">